pilot/pkg/networking/apigen/apigen.go

```diff
@@ -170,6 +170,12 @@ func (g *APIGenerator) Generate(proxy *model.Proxy, push *model.PushContext, w *
 	return resp, model.DefaultXdsLogDetails, nil
 }
 
+func (g *APIGenerator) GenerateDeltas(proxy *model.Proxy, push *model.PushContext, updates *model.PushRequest,
+	w *model.WatchedResource) (model.Resources, []string, model.XdsLogDetails, error) {
+	res, logs, err := g.Generate(proxy, push, w, updates)
+	return res, make([]string, 0), logs, err
```

Comment:
What is the second return value? What does empty []string indicate?

Reply:
I see it is list of deleted resources ? Do we plan to implement it later? If yes, can you please leave a TODO here?

Reply:
The same comment applies for all other than CdsGenerator where we have actually implementation?

Reply:
yeah, []string indicates that no resources were removed, which is what we want. I'm not sure if we should add a TODO because we may not want to support delta for resoures other than LDS and CDS.

With the rest of the resources, we just fall back on the SOTW implementation, and just use the `resources` field in the `DeltaDiscoveryResponse`, with nothing set in `removed_resources` (hence returning an empty slice)

Reply:
Ok. Makes sense. Slight preference to return `[]string{}` instead of make. but it is a huge nit.